Maple2.Server.Game/Session/GameSession.cs

```diff
@@ -451,6 +464,96 @@ public sealed partial class GameSession : Core.Network.Session {
             Player.LastGroundPosition = stateSync.Position;
         }
 
+        bool UpdateStateSyncTracking(ActorState state) {
+            if (StateSyncDistanceTracking.TryGetValue(state, out float totalDistance)) {
+                totalDistance += syncDistance;
+                // 150f = BLOCK_SIZE = 1 meter
+                if (totalDistance >= 150F) {
+                    StateSyncDistanceTracking[state] = 0f;
+                    return true;
+                }
+                StateSyncDistanceTracking[state] = totalDistance;
+                return false;
+            }
+            StateSyncDistanceTracking[state] = syncDistance;
+            return false;
+        }
+
+        bool UpdateStateSyncTimeTracking(ActorState state) {
+            if (StateSyncTimeTracking.TryGetValue(state, out long totalTime)) {
+                totalTime += syncTick;
+                if (totalTime >= 1000) {
+                    StateSyncTimeTracking[state] = 0;
+                    return true;
+                }
+                StateSyncTimeTracking[state] = totalTime;
+                return false;
+            }
+            StateSyncTimeTracking[state] = syncTick;
+            return false;
+        }
```

Comment:
Is this right? I would think there's only 1 ActorState a user can be in. Whenever they change states, the totalTime resets. There wouldn't be a need to have a dictionary because of this.

Reply:
Would we want to ignore any time <1s then? for example jumping up a ladder will rapidly swap you between jump & ladder actorstate
Might not matter in the long run.